binding-parent/datastore-specific-descriptor/src/main/resources/assemblies/datastore-specific-assembly.xml

```diff
@@ -0,0 +1,57 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+  <id>dist</id>
+  <includeBaseDirectory>true</includeBaseDirectory>
+  <baseDirectory>ycsb-${artifactId}-${version}</baseDirectory>
+  <files>
+    <file>
+      <source>README.md</source>
+      <outputDirectory></outputDirectory>
+    </file>
+  </files>
+  <fileSets>
+    <fileSet>
+      <directory>..</directory>
+      <outputDirectory></outputDirectory>
+      <fileMode>0644</fileMode>
+      <includes>
+        <include>CHANGELOG</include>
+        <include>LICENSE.txt</include>
+        <include>NOTICE.txt</include>
+      </includes>
+    </fileSet>
+    <fileSet>
+      <directory>../bin</directory>
```

Comment:
What do you think about adding an `<include>ycsb</include>` here?  I had some backups from merging in `bin/`, which got carried into the distribution.